enarx-keep-sev-shim/src/main.rs

```diff
@@ -14,38 +14,56 @@
 #[cfg(test)]
 fn main() {}
 
+#[macro_use]
+extern crate lazy_static;
+
 pub mod addr;
 pub mod asm;
 pub mod gdt;
 pub mod hostcall;
 pub mod no_std;
+#[macro_use]
 pub mod print;
 pub mod syscall;
 pub mod usermode;
 
-use addr::ShimVirtAddr;
-use core::convert::TryFrom;
 use enarx_keep_sev_shim::BootInfo;
-use hostcall::HostCall;
-use memory::{Address, Page};
+use memory::Page;
+use spinning::RwLock;
 use x86_64::VirtAddr;
 
+static BOOT_INFO: RwLock<Option<BootInfo>> =
+    RwLock::<Option<BootInfo>>::const_new(spinning::RawRwLock::const_new(), None);
```

Comment:
Why are you keeping this as a global?

Reply:
Because some of the lazy_static's need it for the initialization

Reply:
Ack, thanks.

If you rebase I'll approve (cus if I approve now when you rebase it'll dismiss my approval anyway)